packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -112,6 +117,8 @@ export class AccountTrackerController extends BaseController<
     this.defaultState = { accounts: {} };
     this.initialize();
     this.getIdentities = getIdentities;
+    this.getSelectedAddress = getSelectedAddress;
```

Comment:
Q: not sure why we keep a private instance instead of just using the one passed as param? Ref is the same, right?

Reply:
I have the same question, I just kept the same logic that was already there.